client/my-sites/stats/post-trends/index.jsx

```diff
@@ -128,35 +127,47 @@ class PostTrends extends Component {
 	};
 
 	render() {
-		const { siteId, query } = this.props;
+		const { siteId, query, translate } = this.props;
 
-		const leftClass = classNames( 'post-trends__scroll-left', {
-			'is-active': this.state.canScrollLeft,
-		} );
+		// const leftClass = classNames( 'post-trends__scroll-left', {
+		// 	'is-active': this.state.canScrollLeft,
+		// } );
 
-		const rightClass = classNames( 'post-trends__scroll-right', {
-			'is-active': this.state.canScrollRight,
-		} );
+		// const rightClass = classNames( 'post-trends__scroll-right', {
+		// 	'is-active': this.state.canScrollRight,
+		// } );
 
 		/* eslint-disable jsx-a11y/click-events-have-key-events, wpcalypso/jsx-classname-namespace */
 		return (
 			<div className="post-trends">
 				{ siteId && <QuerySiteStats siteId={ siteId } statType="statsStreak" query={ query } /> }
-				<SectionHeader label={ this.props.translate( 'Posting activity' ) } />
-				<Card>
+
+				<div>
+					<div className="post-trends__heading">
+						<h1 className="post-trends__title">{ translate( 'Posting activity' ) }</h1>
+
+						{ /* This will be finished later */ }
+						{ /* <YearNavigation
+							disablePreviousArrow={ false }
+							disableNextArrow={ false }
+							onYearChange={ () => {} }
+						/> */ }
+					</div>
+
+					{ /* 
 					<div className={ leftClass } onClick={ this.scrollLeft } role="button" tabIndex="0">
 						<span className="left-arrow" />
 					</div>
 					<div className={ rightClass } onClick={ this.scrollRight } role="button" tabIndex="0">
 						<span className="right-arrow" />
-					</div>
+					</div> */ }
```

Comment:
Like above, let's delete instead of commenting out unnecessary code.

Reply:
Done!